libraries/matchers.rb

```diff
@@ -1,26 +1,26 @@
 if defined?(ChefSpec)
 
-  ChefSpec.define_matcher :haproxy
-  ChefSpec.define_matcher :haproxy_lb
-  ChefSpec.define_matcher :haproxy_config
+  ChefSpec.define_matcher :haproxy_haproxy
```

Comment:
Yeah, that bit doesn't match, I couldn't get it matching with  `:haproxy` lord knows why.